04-cs-fundamentals/classroom/maths-exercises/essential-maths-exercises.md

```diff
@@ -0,0 +1,282 @@
+# Essential Mathematics - practice problems
+_CS fundamentals 0_
+
+## Overview
+Complete each problem manually (i.e. using paper and pencil) and tally your solution with the one provided for each problem below.
+
+### Sorted items
+For each of the following list of objects, share whether the objects are: _sorted in ascending order_, _sorted in descending order_, _unsorted_.
+
+1. 
+[**2, 4, 6, 8, 10**]
+<details><summary>click here to check your answer</summary>
+These numbers are _sorted in ascending order_.
+</details>
+</br>
+
+2. 
+[**-123, -42, -21, -9, 150, 2100**]
+<details><summary>click here to check your answer</summary>
+These numbers are _sorted in ascending order_.
+</details>
+</br>
+
+3. 
+[**"oranges", "kiwis", bananas", "apples"**]
+<details><summary>click here to check your answer</summary>
+These strings are _sorted in alphabetically descending order_.
+</details>
+</br>
+
+4. 
+[**-3, 705, "apples", 3.14**]
+<details><summary>click here to check your answer</summary>
+This is a list of _unsorted objects_.
+</details>
+</br>
+
+5. 
+[**32, 64, 16, 8, 128**]
+<details><summary>click here to check your answer</summary>
+These numbers are _unsorted_.
+</details>
+</br>
+
+6. 
+[**320, 230, 30, 20, 0**]
+<details><summary>click here to check your answer</summary>
+These numbers are _sorted in descending order_.
+</details>
+</br>
+
+### Maximum and Minimum
+For each of the following list of numbers, what is the **maximum value** and **minimum value**?
+
+1. 
+[**91, 42, 128, 47, 256, 19, 9, 254**]
+<details><summary>click here to check your answer</summary>
+Maximum value: **256**</br>
```

Comment:
Same as above: It seems like on my side, in the preview on GitHub, I see the asterisks, and not the bold formatting. If it's correct on your side, then that works for me though...

Reply:
You're right. I'll get rid of formatting inside the details' tags for now.